lichess-bot.py

```diff
@@ -45,7 +45,8 @@
 
 logger = logging.getLogger(__name__)
 
-with open("lib/versioning.yml") as version_file:
+# with open("lib/versioning.yml") as version_file:
+with open("/mnt/data/lichess_bot_eval_part_2/lichess-bot/lib/versioning.yml") as version_file:
```

Comment:
revert back to the old one? if the old one is not working, then use the absolute path `Path('lib/versioning.yml').absolute()`